packages/svelte-vscode/src/extension.ts

```diff
@@ -111,22 +111,35 @@ export function activateSvelteLanguageServer(context: ExtensionContext) {
     // Add --experimental-modules flag for people using node 12 < version < 12.17
     // Remove this in mid 2022 and bump vs code minimum required version to 1.55
     const runExecArgv: string[] = ['--experimental-modules'];
-    let port = runtimeConfig.get<number>('port') ?? -1;
+
+    const runtimeArgs = runtimeConfig.get<string[]>('runtime-args');
+    if (runtimeArgs !== undefined) {
+        runExecArgv.push(...runtimeArgs);
+    }
+
+    const debugArgs = ['--nolazy'];
+
+    const port = runtimeConfig.get<number>('port') ?? -1;
     if (port < 0) {
-        port = 6009;
+        debugArgs.push('--inspect=6009');
     } else {
         console.log('setting port to', port);
         runExecArgv.push(`--inspect=${port}`);
     }
-    const debugOptions = { execArgv: ['--nolazy', '--experimental-modules', `--inspect=${port}`] };
+
+    debugArgs.push(...runExecArgv);
 
     const serverOptions: ServerOptions = {
         run: {
             module: serverModule,
             transport: TransportKind.ipc,
-            options: { execArgv: runExecArgv }
+            options: { execArgv: runExecArgv, cwd: rootPath }
```

Comment:
Why's this needed?

Reply:
I put it here while experimenting, thinking it might solve a problem I had. It doesn't actually seem necessary.

To elaborate, the crucially important part of this PR is being able to set the flag `--loader ./.pnp.loader.mjs` and that path is supposed to be relative to the workspace root, or in other words relative to `rootPath`. As before mentioned I've never had problems with it, I just left it in because I was paranoid that it was possible to hit some edge case where the cwd would be problematic.

Reply:
Given that this might somehow badly influence other setups I'd be in favor removing it and revisit this in case something comes up

Reply:
Alright! Fixed.